### PR TITLE
Adding responsive web page with bootstrap 4

### PR DESCRIPTION
**Pull request by kots14**
**Issue:** https://github.com/KamandPrompt/baat-cheet/issues/66#issue-713835681

Added responsive home page with some style changes to the home page. Also tried to arrange the ejs templates for the project.

**Summary of changes**

- `container` class in style.css renamed to `app-container` in order to avoid conflict with bootstrap's `container` class
- Separated the headers and footers in `index.ejs` and added bootstrap imports in them
- Added some code in style.css to style the home page

**Note :** 

- [x] Tested responsiveness of home page in Google Chrome
- [x] Tested responsiveness of home page in Mozilla Firefox

**Update:**
- As mentioned [here](https://github.com/KamandPrompt/baat-cheet/pull/70#issuecomment-703033530), fixed the bugs in the PR. Renamed `container` to `app-container` in `client.js` to fix the issue.

### CODE REVIEW

**Comment by LalitNM** [comment]
I would like to point out some bugs, which this PR creates:

- After adding a new room, the screen overlay is not removed automatically.
steps to reproduce: try adding a new room
- The top penal in any new room is not displayed.
steps to reproduce: add a new room and look for top penal(which includes the room name and information about online users)

In addition to these you need to look again the responsiveness of entry page, you need to improve this.

**Comment by kots14** [comment]
I'm trying to fix these bugs. I'll get back to you if I need any help.

**Comment by kots14** [comment]
I've updated the PR and issues seems to be fixed. @LalitNM please let me know if there's anything I missed in the PR that has to be worked on.

**Comment by LalitNM** [comment]
This PR can be further improved by

- removing unnecessary styling code from `style.css`.

- adding a `text-center` class to the `#user` division.

- adding a `font-size` attribute to the error message which comes when two users try to enter with same name.

@kots14 when you will do the above changes, we can merge this.

**Comment by kots14** [comment]
> This PR can be further improved by
> 
> * removing unnecessary styling code from `style.css`.
> * adding a `text-center` class to the `#user` division.
> * adding a `font-size` attribute to the error message which comes when two users try to enter with same name.
> 
> @kots14 when you will do the above changes, we can merge this.

@LalitNM First and Second requirements done. I'm confused on the 3rd requirement. Currently the `font-size` for `#error_response` is `1.3vw`. Shall I change it to a new value?

**Comment by LalitNM** [comment]
> > This PR can be further improved by
> > 
> > * removing unnecessary styling code from `style.css`.
> > * adding a `text-center` class to the `#user` division.
> > * adding a `font-size` attribute to the error message which comes when two users try to enter with same name.
> > 
> > @kots14 when you will do the above changes, we can merge this.
> 
> @LalitNM First and Second requirements done. I'm confused on the 3rd requirement. Currently the `font-size` for `#error_response` is `1.3vw`. Shall I change it to a new value?

Change that in `rem` units, and decide which value you want to give it to, maybe 1 or 0.5 can work.

**Comment by kots14** [comment]
Updated the `font-size` to 1rem for the `#error_response`.
@LalitNM Let me know if there's any other scope of improvement in this PR.

**Comment by LalitNM** [comment]
> @LalitNM Let me know if there's any other scope of improvement in this PR.

 Few more changes

- change `font-size` of title.

- add jumbotron again which you have removed in last commit.

**Comment by kots14** [comment]
For Jumbotron what's the background color and foreground color you'd like? Shall I keep it default?
Also for title is this the snippet you expect me to change?
```
 #bc {
    font-size: 16vw !important;
  }
```
What's the expected font size?

File: public/style.css [hunk 1]
```diff
@@ -25,11 +25,6 @@ li {
   margin: auto;
 }
 
-#user .home-label {
-  background: linear-gradient(#000,  #00b0ff);
-  border: 2px solid  #00b0ff;
-}
-
```
Comment (LalitNM) [inline public/style.css]: @kots14 this is what I am talking about for jumbotron, you can add this code in your next commit.

**Comment by LalitNM** [comment]
> #bc {
>    font-size: 16vw !important;
>  }
>
>What's the expected font size?

You can add font-size in `rem` and check which font-size looks good, I just want to remove font-size based on viewport-width.

File: public/style.css (hunk 1 above)
Comment (kots14) [reply public/style.css]: got it.

**Comment by kots14** [comment]
@LalitNM I've updated the PR. You can check and let me know if there are any other changes required.

**Comment by LalitNM** [comment]
Merged :rocket: Good work @kots14.

File: public/style.css [hunk 2]
```diff
@@ -25,6 +25,11 @@ li {
   margin: auto;
 }
 
+#user .home-label {
+  background: linear-gradient(#000,  #00b0ff);
```
Comment (abhigyank) [inline public/style.css:29]: I dont suppose this blue gradient is looking good. Doesn't the pure black look better?

Comment (LalitNM) [reply public/style.css:29]: Pure black was fine but I thought that we should highlight the useful area by making a blue gradient.

Comment (abhigyank) [reply public/style.css:29]: Okay, understood. In my opinion, doesn't look very aesthetically pleasing.

Comment (kots14) [reply public/style.css:29]: With the current file, I can make it look like option 1 or 2 with a small commit. I can raise a new Issue on this repository and make this change according to that.

1. image
2.  image

However, if it doesn't fit the requirement, please let me know when you raise a new Issue specifying the required look for this. I'd like to work on that.

Comment (abhigyank) [reply public/style.css:29]: @kots14 Let's go with 2. Since Lalit wanted to insert a blue shade. Can you make a PR for this?

Comment (LalitNM) [reply public/style.css:29]: One more problem this PR creates is the git history. After merging this PR the git history is clumsy. I am gonna revert all changes done in this PR and will merge your next PR with the changes requested by @abhigyank and these changes. Make sure to that you have added all these changes in new PR. If you need the present state of repository for any reference then check [70-merged](https://github.com/KamandPrompt/baat-cheet/tree/70-merged) branch @kots14.

Comment (kots14) [reply public/style.css:29]: @LalitNM Thanks for creating the reference branch. I'll raise a new PR with the changes mentioned by @abhigyank.

Comment (kots14) [reply public/style.css:29]: Added new PR https://github.com/KamandPrompt/baat-cheet/pull/82 for https://github.com/KamandPrompt/baat-cheet/issues/66

**Pull request merged**